tensorflow-xla-runtime.spec

```diff
@@ -9,11 +12,16 @@ BuildRequires: cmake
 cp -r ${PY3_TENSORFLOW_ROOT}/lib/python%{cms_python3_major_minor_version}/site-packages/tensorflow .
 
 %build
-
+source %{_sourcedir}/env
 export CPATH="${CPATH}:${EIGEN_ROOT}/include/eigen3"
 
+CXXFLAGS="-fPIC %{arch_build_flags} ${CMS_EIGEN_CXX_FLAGS}"
+%ifarch x86_64
+    CXXFLAGS="${CXXFLAGS} -msse3"
+%endif
```

Comment:
I guess adding `-DEIGEN_NEON_GEBP_NR=4` for aarch64 should fix this

Reply:
or better to update scram-tools.file/tools/eigen/env.file and add this flagfor aarch64 there